doc/pod_chaos.md

```diff
@@ -39,6 +40,7 @@ Description:
 
 * **action** defines the specific chaos action for the pod. In this case, it is pod failure.
 * **mode** defines the mode to run chaos action. Supported mode: `one / all / fixed / fixed-percent / random-max-percent`.
+* **value** depends on the value of `mode`. If `mode` is `one` or `all`, leave `value` empty. If `fixed`, provide an integer of pods to do chaos action. If `fixed-percent`, provide a number from 0-100 to specify the max % of pods the server can do chaos action. If `random-max-percent`, provide a number from 0-100 to specify the % of pods to do chaos action
```

Comment:
" the max % of " and " the % of " looks strange ,  is it inappropriate?

Reply:
It was copied from PodChaos field comment. I have changed it to percent.